src/ontology/mondo-ingest.Makefile

```diff
@@ -172,20 +172,34 @@ update-jinja-sparql-queries: python-install-dependencies
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
-config/%_term_exclusions.txt config/%_exclusion_reasons.robot.template.tsv: config/%_exclusions.tsv component-download-%.owl $(REPORTDIR)/mirror_signature-%.tsv $(REPORTDIR)/component_signature-%.tsv metadata/%.yml python-install-dependencies
+#################
+### Exclusions ##
+#################
+# Exclusions: by ontology
+# TODO: add prereqs back
+# $(REPORTDIR)/%_term_exclusions.txt $(REPORTDIR)/%_exclusion_reasons.robot.template.tsv: config/%_exclusions.tsv component-download-%.owl $(REPORTDIR)/mirror_signature-%.tsv $(REPORTDIR)/component_signature-%.tsv metadata/%.yml python-install-dependencies
+$(REPORTDIR)/%_term_exclusions.txt $(REPORTDIR)/%_exclusion_reasons.robot.template.tsv:
```

Comment:
@matentzn I found out why this was producing 0 results. It is because the [omim_exclusions.tsv.zip](https://github.com/monarch-initiative/mondo-ingest/files/9312932/omim_exclusions.tsv.zip) has empty columns for `exclude_children` and `exclusion_reason`.

If you want, I can (a) throw an error if something like this happens, or (b) do some default behavior.

@sabrinatoro @nicolevasilevsky Is there a single Google Drive folder that has all of the exclusion table Google Sheets? If so, can you give me permission and link me? If you guys have finished this for OMIM, I can download the results and run this again. Otherwise, I think we will probably need this filled out some time soon.